apps/api/src/database/schema.prisma

```diff
@@ -433,19 +433,22 @@ model DocumentActivityLog {
 }
 
 model S51Advice {
-  id             Int      @id @default(autoincrement())
-  caseId         Int
-  title          String
-  firstName      String?
-  lastName       String?
-  enquirer       String?
-  enquiryMethod  String
-  enquiryDate    DateTime
-  enquiryDetails String
-  adviser        String
-  adviceDate     DateTime
-  adviceDetails  String
-  createdAt      DateTime @default(now())
-  updatedAt      DateTime @default(now())
-  case           Case     @relation(fields: [caseId], references: [id])
+  id              Int      @id @default(autoincrement())
+  caseId          Int
+  title           String
+  firstName       String?
+  lastName        String?
+  enquirer        String?
+  enquiryMethod   String
+  enquiryDate     DateTime
+  enquiryDetails  String
+  adviser         String
+  adviceDate      DateTime
+  adviceDetails   String
+  referenceNumber Int      @default(00001)
+  redactedStatus  String   @default("not_redacted")
```

Comment:
in DocumentVersion, redactedStatus allows null, and no default value - so only changes if the user explicitly sets it to redacted, or not_redacted.  Should we be consistent and do the same thing here?